.env

```diff
@@ -0,0 +1 @@
+DockerImageVersion=''
```

Comment:
Remove the .env file and ignore it in your .gitignore file and .dockerignore file as well